setup.py

```diff
@@ -31,7 +31,7 @@ def get_version(short=False):
       url='https://github.com/williamjameshandley/anesthetic',
       packages=find_packages(),
       scripts=['scripts/anesthetic'],
-      install_requires=['numpy', 'scipy', 'pandas', 'matplotlib'],
+      install_requires=['numpy', 'scipy', 'pandas<=1.0.5', 'matplotlib'],
```

Comment:
You've added the `<=1.0.5` flag to both `requirements*.txt` and to `setup.py`. Should that generally be done? Should we do this for `matplotlib` and `scipy` also?

Reply:
>You've added the <=1.0.5 flag to both requirements*.txt and to setup.py. Should that generally be done? 

There are [ways round the duplication](https://stackoverflow.com/questions/43658870/requirements-txt-vs-setup-py) inherent in having dependencies in both requirements.txt and setup.py, but they're always very fiddly, and just as liable to human error. For now it's simplest/least confusing for users to maintain both, irritating though that is

>Should we do this for matplotlib and scipy also?

Yes : ) Also synced requirements_extras.txt for compleness